LICENSE

```diff
@@ -1,3 +1,29 @@
+MIT License applies only to code snippets
```

Comment:
The Creative Commons Attribution-ShareAlike 4.0 International applies to the everything. Code snippits are, additionally, available under the MIT License.